components/automate-cs-bookshelf/habitat/hooks/run

```diff
@@ -24,7 +26,13 @@ pg-helper create-extension "$DBNAME" "uuid-ossp"
 pg-helper sqitch-deploy "$DBNAME" "{{pkgPathFor "chef/bookshelf"}}/schema"
 pg-helper fix-permissions "$DBNAME"
 
-secrets-helper generate bookshelf.access_key_id 16 --if-not-exists
-secrets-helper generate bookshelf.secret_access_key 16 --if-not-exists
+if [[ -z "${CURRENT_SECRET_KEY}" ]]; then
+  secrets-helper generate bookshelf.access_key_id 64 --if-not-exists
```

Comment:
As discussed, `--if-not-exists` might work in the fresh installtions but fails in case of moving from AWS S3 to bookshelf(filesystem). Could you please verify this scenario for making sure it does correct.

Reply:
@YashviJain01  Please reply to the comment before resolving the conversation